src/app/workflow/version-modal/version-modal.component.html

```diff
@@ -93,7 +100,7 @@ <h4 mat-dialog-title>{{ isPublic || !canWrite ? 'View' : 'Edit' }} {{ (isService
                   title="Remove test parameter file"
                   type="button"
                   class="btn btn-default form-sm-button"
-                  *ngIf="!isPublic"
+                  *ngIf="!isPublic && !version.frozen && !version.doiURL"
```

Comment:
`!version.doiURL` seems redundant with `!version.frozen`
(all doi versions should have been frozen first)